src/event-log/event-log.ts

```diff
@@ -29,5 +34,12 @@ export interface EventLog {
    * @param tenant
    * @param watermark
    */
-  getEventsAfter(tenant: string, watermark?: string): Promise<Array<Event>>
+  getEvents(tenant: string, options?: GetEventsOptions): Promise<Array<Event>>
+
+  /**
+   * deletes any events that have any of the cids provided
```

Comment:
Add documentation on what the returned `number` is for implementors.

Reply:
thanks